examples/SendValuesTCP/SendValuesTCP.ino

```diff
@@ -25,23 +27,24 @@ Ubidots ubidots(UBIDOTS_TOKEN, UBI_TCP);
 /****************************************
  * Main Functions
  ****************************************/
-
 void setup() {
   Serial.begin(115200);
-  ubidots.wifiConnect(WIFI_SSID, WIFI_PASS);
-  // ubidots.setDebug(true);  // Uncomment this line for printing debug messages
+  Ubidots::wifiConnect(WIFI_SSID, WIFI_PASS);
+  ubidots = new Ubidots(UBIDOTS_TOKEN, UBI_TCP);
+  //ubidots->setDebug(true); //Uncomment this line for printing debug messages
```

Comment:
For the sake of styling in all comment => "// <your comment>"

Reply:
Solved in the latest commit